src/app/services/chat.service.ts

```diff
@@ -57,4 +61,8 @@ export class ChatService {
     );
     return chatsRef;
   };
+
+  sendMessage = (message: MessageI) => {
+    this.testMessages$.next(this.testMessages$.getValue().concat([message]));
```

Comment:
Here, we can add the id in, so we don't need AngularFirestore imported into the component (would generally try to avoid that and keep it in the services instead)
```suggestion
    message.id = this.afs.createId();
    this.testMessages$.next(this.testMessages$.getValue().concat([message]));
```